src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -3567,6 +3559,11 @@ internal TElement[] ToArray()
         }
     }
 
+    // NOTE: DO NOT DELETE THE FOLLOWING DEBUG VIEW TYPES.
+    // Although it might be tempting due to them not be referenced anywhere in this library,
+    // Visual Studio currently depends on their existence to enable the "Results" view in 
+    // watch windows.
+
```

Comment:
Is the lack of disposal of enumerators in `Items` for a reason? It looks wrong, but it also looks deliberate, like the sort of wrong-seeming thing that is actually right.